packages/api/cms-api/src/common/filter/many-to-one.filter.ts

```diff
@@ -0,0 +1,20 @@
+import { Field, InputType } from "@nestjs/graphql";
+import { IsOptional, IsString } from "class-validator";
+
+@InputType()
+export class ManyToOneFilter {
+    @Field(() => [String], { nullable: true })
```

Comment:
Should we use the `ID` scalar here to show that IDs should be used?

Reply:
yes, good idea